### PR TITLE
Chapter10 2

### PR DESCRIPTION
**Pull request by fgazzelloni**

### CODE REVIEW

**Review by jonthegeek — APPROVED** [review]
Looks good as long as it builds. Thanks!

File: 10_dynamic_regression_models.Rmd [hunk 1]
```diff
@@ -217,8 +217,137 @@ insurance %>%
...
+Create a newdata with sampled TVadverts:
+```{r}
+insurance_future2 <- new_data(insurance, 20) |>
+  mutate(TVadverts = sample)
+insurance_future2%>%head
+```
+
+Fit the model on sampled TVadverts and plot it:
+```{r}
+fit_best |>
```
Comment (jonthegeek) [inline 10_dynamic_regression_models.Rmd:251]: Starting here you reference `fit_best`, but it doesn't exist. Is it supposed to have been created at some point above here? The book won't build with failing code.

**Review by jonthegeek — CHANGES_REQUESTED** [review]
There's an error on line 244.

File: 10_dynamic_regression_models.Rmd (hunk 1 above)
Comment (fgazzelloni) [reply 10_dynamic_regression_models.Rmd:251]: updated chapter

**Review by jonthegeek — APPROVED** [review]
Thanks!

**Pull request merged**